src/plays/inspirational-quotes/type/CardProps.tsx

```diff
@@ -0,0 +1,5 @@
+export type CardProps = {
```

Comment:
@muzaffar640 , I would suggest creating one type file with the `.ts` extension and defining all your types there.
Then you can use types from a single file.
Also, one thing to note, if we are not returning any JSX from the file then file extensions should be either `.js` or `.ts`